ietf/api/views.py

```diff
@@ -202,6 +203,25 @@ def err(code, text):
             else:
                 send_account_creation_email(request, email)
                 response += ", Email sent"
+
+            # handle nomcom volunteer
+            if data['is_nomcom_volunteer'] and object.person:
+                now = datetime.datetime.now()
+                month = now.month
+                year = now.year
+                if month > 7:
```

Comment:
A comment explaining this if statement would be helpful.

Reply:
Added comment